gef-ui/templates/partials/facility-value.njk

```diff
@@ -121,7 +121,10 @@
             classes: "govuk-label--m"
           },
           hint: {
-            text: "This is the ongoing percentage you’ll charge the exporter for providing the cash facility. It does not include any upfront fees you charge, such as arrangement fees."
+            text: "This is the annual margin charged as a percentage to the exporter for providing the " + facilityTypeString + " facility. It does not include any upfront fees you charge, such as arrangement fees.",
+            attributes: {
+              'data-cy' : 'interest-percentage-hint'
```

Comment:
Very minor nit pick

```suggestion
              'data-cy': 'interest-percentage-hint'
```

Reply:
Thanks Tony! Fixed the formatting as suggested